app/tests/components/TimeTravel.test.tsx

```diff
@@ -48,7 +48,7 @@ describe('The TimeTravel component', () => {
     fireEvent.mouseDown(datePicker);
     fireEvent.change(datePicker, { target: { value: "2020-01-02" } });
     await waitFor(() => {
-      // difference due to timezone
+      // difference due to timezone 
```

Comment:
If we set the default time zone in this test, we won't need this one-day difference

Reply:
For some reason test is failing
```  <input
                class="TimeTravel__StyledDatePicker-sc-wl9pxw-3 kIPtRb form-control"
                placeholder="YYYY-MM-DD"
                type="text"
                value="2020-01-01"
              />
            </div>
          </div>
        </div>
      </div>
    </body>

      48 |     fireEvent.mouseDown(datePicker);
      49 |     fireEvent.change(datePicker, { target: { value: "2020-01-02" } });
    > 50 |     await waitFor(() => {
         |           ^
      51 |       expect(screen.getByText(`Current date is: 2020-01-02`)).toBeInTheDocument();
      52 |     }); 
      53 |   });
```
while on local machine it passes

Reply:
Do we need to explicitly set the Luxon settings in this test?

Reply:
Fixed by changing how we parse Date in TimeTravel (instead of using dateFormat use luxon .fromJSDate